### ISSUE TITLE
Continue Search

### ISSUE THREAD

**Issue by FelixNeutatz** [automl/auto-sklearn#885]
Dear all,

I am currently experimenting with auto-sklearn and I am trying to get how the progress of auto-sklearn is across time:

```
times = list(range(60, 60*61, 60))
test_accuracies = []
for time in times:
    automl = autosklearn.classification.AutoSklearnClassifier(time_left_for_this_task=time,
                                                              n_jobs=4,
                                                              resampling_strategy='cv',
                                                              resampling_strategy_arguments={'folds': 5}
                                                              )

    feat_type = ['Categorical' if ci else 'Numerical' for ci in categorical_indicator]
    automl.fit(X_train, y_train, metric=roc_auc_scorer, feat_type=feat_type)
    automl.refit(X_train, y_train)
    
    test_accuracies.append(auc(automl, X_test, y_test))
```
However, this is highly inefficient. So, I would like to implement the same in this way:
Run AutoSklearn for 1 minute, check the test accuracy and then continue the search - instead of starting from scratch.
Is there any way to continue the search?

Thank you for your help.

Best regards,
Felix

**Comment by mfeurer** [comment]
Yes, this should be possible. However, we haven't tested this together with the n_jobs features, so I suggest that you use Auto-sklearn's manual multiprocessing setup as shown [in this example](https://automl.github.io/auto-sklearn/master/examples/example_parallel_manual_spawning.html).

Continuing to search is also handled with the manual multiprocessing setup: if you start Auto-sklearn again (make sure to use a different seed) with `shared=mode=True` and the same temporary directory it will pick up the old results and continue the search. Please also make sure to use no meta-learning and a random initial configuration when continuing the search as shown in the example.

**Comment by FelixNeutatz** [comment]
Thank you for your help. I will try it :)
